index.js

```diff
@@ -1,106 +1,151 @@
 const path = require("path");
 const { execSync } = require("child_process");
 const fs = require("fs");
-const axios = require('axios');
-const FormData = require('form-data');
-const converter = require('json-2-csv');
-const split = require('split');
-const { finished } = require('stream');
+const axios = require("axios");
+const FormData = require("form-data");
+const converter = require("json-2-csv");
+const split = require("split");
+const { finished } = require("stream");
 // Constructing promisify from util
-const { promisify } = require('util');
+const { promisify } = require("util");
 // Defining finishedAsync method
 const finishedAsync = promisify(finished);
 
+const eventEndTimeLimit = 5 * 60 * 1000; // after this time limit, events are sent even if not ended
+
 function initialize(config, logger) {
-    this.config = config;
-    this.logger = logger;
-    this.config.logFilename = 'active.log';
+  this.config = config;
+  this.logger = logger;
+  this.config.logFilename = "active.log";
```

Comment:
Move as constant, never change configuration inside of the module. use LOG_FILENAME